# Name of the frame that the data was collected in
string frame

# One point group for observations of the same landmark
PointGroup[] point_groups